README.md

```diff
@@ -1,39 +1,125 @@
+The Protocol Labs Research Grant Program aims to both support collaborative work on problems defined by the broader research community and drive progress on clearly scoped research projects critical to our work at Protocol Labs. We encourage proposals aligned with PL's mission to drive breakthroughs in computing to push humanity forward, and we welcome unique perspectives and diverse backgrounds.
 
-*Proposals are considered and awarded based on individual objectives. We may choose to fund all objectives within a proposal or limit our funding to those most relevant and promising. Please be clear when describing your proposal's objectives and how you plan to achieve them.*
+## Funding instruments
 
+Protocol Labs Research funds three broad categories of grants:
+1. Topic-specific Requests for Proposals (RFPs), designed to fund goal-driven research in response to specific open problems defined by Protocol Labs (PL). RFPs are posted intermittently based on need and readiness of our research labs to support these collaborations.
+2. Open research grants, intended to support researchers pursuing topics of interest to Protocol Labs. These include:
+    * Doctoral Fellowships
+    * Postdoctoral Fellowships
+    * Faculty Summer Research Grants
+    * Faculty Research Sabbatical Awards
+3. A wildcard Request for Proposals, RFP-X, by which we fund research proposed by applicants where funding from one of our other grant structures does not readily apply. Note that we have a strong preference for funding via the open grants framework, and **RFP-X applications that could be handled within the existing framework will be rejected without review.**
 
-# What's in This Repo?
+Evaluation of applications for the open grants will occur in quarterly rounds. Decisions will normally be made and communicated to applicants within eight weeks of these submission deadlines. Evaluation for RFPs takes place according to the timeline in the call. RFP-X applications are evaluated year-round but usually have longer decision times.
 
-This repo contains Requests For Proposals (RFPs) for the Grant program, which is elaborated below.
+If you have any questions, please email us at **research@protocol.ai**.
```

Comment:
Is this preferred over the research acceleration address?

Reply:
```suggestion
If you have any questions, please email us at **research-grants@protocol.ai**.
```